<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AntEditorWidget</class>
 <widget class="QWidget" name="AntEditorWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>793</width>
    <height>587</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QToolButton" name="editButton">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset theme="edit-select-all"/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="insertButton">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset theme="list-add"/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox">
       <property name="minimumSize">
        <size>
         <width>180</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="3" column="0">
    <widget class="VectorialView" name="graphicsView">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>2</horstretch>
       <verstretch>2</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="0" column="1" rowspan="4">
    <widget class="QToolBox" name="toolBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>1</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="AntShapeTypeEditorWidget" name="shapeTypeEditor">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>256</width>
        <height>509</height>
       </rect>
      </property>
      <attribute name="label">
       <string>Shape Types</string>
      </attribute>
     </widget>
     <widget class="MeasurementTypeWidget" name="measurementTypeEditor">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>256</width>
        <height>509</height>
       </rect>
      </property>
      <attribute name="label">
       <string>Measurement Types</string>
      </attribute>
     </widget>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QTreeView" name="treeView">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>1</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>VectorialView</class>
   <extends>QGraphicsView</extends>
   <header location="global">fort-studio/widget/vectorgraphics/VectorialView.hpp</header>
  </customwidget>
  <customwidget>
   <class>AntShapeTypeEditorWidget</class>
   <extends>QWidget</extends>
   <header location="global">fort-studio/widget/AntShapeTypeEditorWidget.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>MeasurementTypeWidget</class>
   <extends>QWidget</extends>
   <header location="global">fort-studio/widget/MeasurementTypeWidget.hpp</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
